app/javascript/components/buildpack/Detail.tsx

```diff
@@ -104,6 +104,12 @@ class Detail extends React.Component<{match: {params: any}}, { buildpack: any }>
                                     </Card.Title>
                                     <code>pack build myapp --buildpack {buildpack.latest.namespace}/{buildpack.latest.name}@{buildpack.latest.version}</code>
                                 </div>
+                                <div>
```

Comment:
@arnav-t looks like you pulled these changes from your previous PR.  Could you rebase off of main?

Reply:
@elbandito Done. Is this good?